package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "ah-frontend-kronos",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack-dev-server --open --hot --port 3000 --mode development",
```

Comment:
Why this configuration of  the `--port 3000` and `--mode`?  I think its not needed during the Heroku deployment, unless you have some different configuration for us during the deployment. Thank you

Reply:
Thanks @Emmanuel-Dominic specifying port and mode is necessary. This because `npm start` will only be used in the development environment if you want to host that is a different environment so you will make use of `npm build`.
Kindly use this to gain a more understanding. https://stackoverflow.com/questions/43664200/what-is-the-difference-between-npm-install-and-npm-run-build